### DISCUSSION TITLE
Spotlight feature

### DISCUSSION THREAD

**Discussion by antaramimi** [Softal-io/presentify-community#2] [Ideas]
The screen would be highlighted where the cursor is and the rest would be blurred.

**Comment by danyindrawan** [comment]
Nice!!!

**Comment by rampatra** [comment]
This is live in Presentify v6.0.0.

![app-store-spotlight](https://github.com/user-attachments/assets/8a7e5465-637b-454b-bd4a-9e4679cbe7f8)
